variant-inc/terraform-variant-apps
# update on configmaps

# Description

Convert multiple config maps for kafka topics to single one for managed and single one for existing.

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

1. Specify this version of terraform-variant-apps in your actions octopus workflow
```
- name: Lazy Action Octopus
        uses: variant-inc/actions-octopus@v3
        with:
          deploy_package_version: 1.0.1-b-cloud-1973-kaf0001.567
```
2. Add kafka infrastructure to your DX deployment yaml
https://github.com/variant-inc/demo-python-flask-variant-api/blob/demo/luka-2/.variant/deploy/luka-api.yaml#L12-L27
3. Deploy the app and infrastructure [GH Actions](https://github.com/variant-inc/demo-python-flask-variant-api/actions/runs/3008313991) & [Octopus](https://octopus.apps.ops-drivevariant.com/app#/Spaces-2/projects/demo-api-luka/deployments/releases/0.1.0-demo-luka-2-0001.452/deployments/Deployments-45034?activeTab=taskSummary)
4. Verify that resources are created as expected and visible in container.
Config maps:
![image](https://user-images.githubusercontent.com/63529189/188903152-84fa10cc-9fe3-488d-9f84-2ccf963e5955.png)
Container ENV:
![image](https://user-images.githubusercontent.com/63529189/188902607-063b1749-8f0b-41af-bda0-3b822261863e.png)



## Checklist
